pkg/api/frontendsettings.go

```diff
@@ -270,6 +271,7 @@ func (hs *HTTPServer) getFrontendSettings(c *contextmodel.ReqContext) (*dtos.Fro
 
 		Azure: dtos.FrontendSettingsAzureDTO{
 			Cloud:                                  hs.Cfg.Azure.Cloud,
+			Clouds:                                 getAzureClouds(hs.Cfg.Azure),
```

Comment:
I didn't use that method because it merges the pre-defined clouds with any custom clouds.  I believe there was a desire to NOT send the list of cloud unless it was a custom list?  If that is correct, then we can't call Clouds() we really just want the custom clouds.

Reply:
Ah yes that makes sense as the frontend SDK already has the predefined clouds. In that case, we can use the `CustomClouds` function directly rather than wrapping it and change the type to `[]azsettings.AzureCloudInfo`. This makes more sense rather than creating a new type solely for this purpose.